### PR TITLE
Appium aliases to properties

### PR DESCRIPTION
**Pull request by ncalaway**
Merging into Appium branch

### CODE REVIEW

File: src/appium/appium-helpers.ts [hunk 1]
```diff
@@ -65,3 +79,99 @@ export const appiumSessionDestroy = (scenario: Scenario) => {
     });
   };
 };
+
+export const appiumFindByUiAutomator = async (
+  response: AppiumResponse,
+  selector: string,
+  text: string,
+  opts?: FindAllOptions | null
```
Comment (johnsickels) [inline src/appium/appium-helpers.ts:87]: DO you need `null` if the parameter is optional?

File: src/appium/appium-helpers.ts [hunk 2]
```diff
@@ -65,3 +79,99 @@ export const appiumSessionDestroy = (scenario: Scenario) => {
...
+      UiSelector = UiSelector + 'text("' + text + '")';
+  }
+
+  const res = await sendAppiumRequest(
+    response.scenario,
+    `/session/${response.sessionId}/elements`,
+    {
+      method: "post",
+      data: {
+        using: "-android uiautomator",
```
Comment (johnsickels) [inline src/appium/appium-helpers.ts:134]: Is this ok as device specific?

**Review by johnsickels — APPROVED** [review]
Looks good. May I request some jsdoc comments around functions that may be not so obvious?

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

File: src/appium/appium-helpers.ts (hunk 2 above)
Comment (ncalaway) [reply src/appium/appium-helpers.ts:112]: Yes, this is Android only.

**Comment by ncalaway** [comment]
> Looks good. May I request some jsdoc comments around functions that may be not so obvious?
> 
> https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

Which functions did you have in mind?

**Review by jasonbyrne — APPROVED** [review]

**Pull request merged**